pkg/sql/opt/exec/execbuilder/testdata/project

```diff
@@ -113,21 +113,16 @@ SELECT u + v FROM (SELECT x + 3, y + 10 FROM t.a) AS foo(u, v)
 exec-explain
 SELECT x, x, y, x FROM t.a
```

Comment:
@RaduBerinde, this PR breaks this test. I added a TODO in `builder.go` for you to add support for the new Presentation property, which will fix this test.